### PR TITLE
[STACK 1] Package name change, `radiotherapyai` -> `rai`

### PR DESCRIPTION
**Pull request by SimonBiggs**
I'll be requesting for a name transfer for the following abandoned project:

https://pypi.org/project/rai/

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ bibtex_bibfiles:
 
 # Information about where the book exists on the web
 repository:
-  url: https://github.com/RadiotherapyAI/regulatory
+  url: https://github.com/RadiotherapyAI/rai
```
Comment (sjswerdloff) [inline docs/_config.yml:24]: rai vs regulatory?

**Review by sjswerdloff — COMMENTED** [review]
Double check rai vs regulatory

File: docs/_config.yml (hunk 1 above)
Comment (SimonBiggs) [reply docs/_config.yml:24]: Yup, that's supposed to be rai now. In fact, https://github.com/RadiotherapyAI/regulatory actually redirects to https://github.com/RadiotherapyAI/rai just to catch old links hanging around the place.

Thanks @sjswerdloff :)

**Review by sjswerdloff — APPROVED** [review]
Clarification sufficient

**Pull request merged**